### DISCUSSION TITLE
Uninstallation steps

### DISCUSSION THREAD

**Discussion by s-tomar** [conda-forge/miniforge#303] [General]
Hi,
Thank you for all the work in creating **miniforge**.
In the uninstallation process, if we run the reverse process (`conda init --reverse`) without first defining `CONDA_BASE_ENVIRONMENT`, then `CONDA_BASE_ENVIRONMENT=$(conda info --base)` doesn't work because it can't find `conda` in the path. To avoid manually typing the full path, we can first execute `CONDA_BASE_ENVIRONMENT=$(conda info --base)`, and the remaining process then follows smoothly.

**Comment by hmaarrfk** [comment]
Sounfs good. If you could submit a PR that would be appreciated!

**Reply by s-tomar** [reply]
Thanks.
But is this issue big/serious enough to create a PR? I guess this (very) minor reordering of the commands can be easily/quickly arranged by someone who has write access to this page. If you still deem it necessary, I'll do it tomorrow.

**Reply by hmaarrfk** [reply]
It's pretty hard to test things like this. So often a PR implies that you tested things to yourself. I can review, but i likely won't run the installation steps but I'll squint to make sure they are ok.

If you can think of a way to test the instructions, that's even better! That all comes with a good PR.

**Reply by s-tomar** [reply]
Thanks. Just created a PR. The only change recommended is to update the README file, which should be self-explanatory to the user.
I'm sorry for the delay. I had to learn/test few things 🙂
